src/ApparentHorizons/Tags.hpp

```diff
@@ -103,10 +110,12 @@ struct InvHessian : db::ComputeTag {
 template <typename Frame>
 struct Radius : db::ComputeTag {
   static std::string name() noexcept { return "Radius"; }
-  SPECTRE_ALWAYS_INLINE static auto function(
-      const ::Strahlkorper<Frame>& strahlkorper) noexcept {
-    return strahlkorper.ylm_spherepack().spec_to_phys(
-        strahlkorper.coefficients());
+  using return_type = DataVector;
+  static void function(const gsl::not_null<DataVector*> radius,
```

Comment:
let's just move the definition of this to the cpp file to avoid including ContainerHelpers.hpp